pages/documentation/migration-V3.md

```diff
@@ -0,0 +1,822 @@
+---
+permalink: /documentation/migration/
+layout: styleguide
+title: Migrating to USWDS 3.0
+lead: Migrate to USWDS 3.0 for Sass Module support and an unbundled design system with minimal upgrade effort
+category: How to use USWDS
+subnav:
+- text: Why migrate to USWDS 3.0?
+  href: '#why-migrate-to-uswds-30'
+- text: How to use this guide
+  href: '#how-to-use-this-guide'
+- text: Migration overview
+  href: '#migration-overview'
+- text: 1. Check your current USWDS code and settings versions
+  href: '#1-check-your-current-uswds-code-and-settings-versions'
+- text: 2. Integrate latest USWDS 2 changes
+  href: '#2-integrate-latest-uswds-2-changes'
+- text: 3. Install the USWDS 3.0 package
+  href: '#3-install-the-uswds-30-package'
+- text: 4. Update your Sass compiler settings and recompile CSS
+  href: '#4-update-your-sass-compiler-settings-and-recompile-css'
+- text: 5. Update to Sass module syntax
+  href: '#5-update-to-sass-module-syntax'
+- text: 6. Find which settings from your theme files you’ve customized.
+  href: '#6-find-which-settings-from-your-theme-files-youve-customized'
+- text: 7. Optimize your installation with component packages
+  href: '#7-optimize-your-installation-with-component-packages'
+---
+
+## Why migrate to USWDS 3.0?
+
+{:.border-top-2px.border-base-lighter.padding-top-1}
+**Use modern Sass syntax before the old syntax loses official support.** USWDS stylesheets are written in a language called Sass. This language updated its syntax in 2020 and the old syntax is now deprecated. Sass will no longer support the old syntax starting in October 2022. We want teams to use source code with reliable support throughout the life of their project.
+
+**Improve performance and reduce the size of your project CSS.** Using USWDS 3.0 with the new Sass syntax — called Sass module syntax — allows teams to unbundle their implementations of USWDS and use only the components and code that they need on their project. Depending on your project, this could mean significant reduction of CSS code you ship. This means faster load times, better scores in performance evaluation tools, and a better developer experience. 
+
+**Stay up-to-date with minimal hassle.** We want teams to benefit from the most current version of USWDS. Many projects should be able to migrate from USWDS 2 to USWDS 3.0 in about an hour or less. This new version makes no markup or style changes from USWDS 2.13.3. If you already use USWDS 2.13.0 or later, you should be able to update to USWDS 3.0 in a matter of minutes. Additionally, the under-the-hood changes we're introducing in USWDS 3.0 will make it easier to stay up-to-date with USWDS over time. An incremental update now will make subsequent updates simpler as well. 
+
+## How to use this guide
```

Comment:
@bonnieAcameron @thisisdano Wondering if this `How to use this guide` instruction would be more useful inside the `Step 2. Integrate latest USWDS 2 changes` section. It seems as though it is instruction that applies only to that step.

Reply:
Hi @amyleadem ! I'm fleshing out this section today. Let's keep it where it is for now.